src/System.Slices/src/System/SpanEqualsExtensions.cs

```diff
@@ -0,0 +1,117 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>look at the SpanExtensionsTemplate.tt</auto-generated>
+//------------------------------------------------------------------------------
+// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+namespace System
+{
+    public static partial class SpanExtensions
+    {
+        /// <summary>
+        /// Determines whether two spans are equal (byte-wise) by comparing the elements by using memcmp
+        /// </summary>
+        /// <param name="first">A span of bytes to compare to second.</param>
+        /// <param name="second">A span of bytes T to compare to first.</param>
+        public static bool SequenceEqual(this Span<byte> first, Span<byte> second)
```

Comment:
I feel like we are lacking some better templates in our language, feels like this should be fixed in the compiler and not in here